services/sleeper/docker-compose-meta.yml

```diff
@@ -2,50 +2,45 @@ services:
   sleeper:
     build:
       labels:
-        io.simcore.authors:
-          '{"authors": [{"name": "Andrei Neagu", "email": "neagu@itis.swiss",
+        io.simcore.authors: '{"authors": [{"name": "Odei Maiz", "email": "maiz@itis.swiss",
+          "affiliation": "IT''IS Foundation"}, {"name": "Andrei Neagu", "email": "neagu@itis.swiss",
           "affiliation": "IT''IS Foundation"}]}'
         io.simcore.contact: '{"contact": "neagu@itis.swiss"}'
-        io.simcore.description:
-          '{"description": "A service which awaits for time
+        io.simcore.description: '{"description": "A service which awaits for time
           to pass, used for system validation."}'
```

Comment:
would remove the "used for system validation part"... you might want to sleep for some other reason

Reply:
done